<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SwitchBranchDialog</class>
 <widget class="QDialog" name="SwitchBranchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="radioButtonExistingBranch">
       <property name="text">
        <string>Existing branch</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxBranchSelect"/>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="radioButtonNewBranch">
       <property name="text">
        <string>New branch</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditNewBranchName">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="checkBoxForce">
        <property name="text">
         <string>Force</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>39</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SwitchBranchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonExistingBranch</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBoxBranchSelect</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonNewBranch</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditNewBranchName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
